gems/quilt_rails/README.md

```diff
@@ -123,3 +128,44 @@ Installs the Node dependencies, provide a basic React app (in TypeScript) and mo
 ### `sewing-kit:install`
 
 Adds a basic `sewing-kit.config.ts` file.
+
+## Advanced use
+
+### Customizing the node server
+
+By default, sewing-kit bundles in `@shopify/react-server-webpack-plugin` for `quilt_rails` applications to get you up and running fast without needing to manually write any node server code. If you would like to customize what data your react application receives from the incoming request, you can add your own `server.js` / `server.ts` file to the app folder.
+
+```
+└── app
+   └── ui
+      └─- index.js
+      └─- server.js
+```
+
+```tsx
+// app/ui/server.tsx
+import 'isomorphic-fetch';
+import {createServer} from '@shopify/react-server';
+import {Context} from 'koa';
+import React from 'react';
+
+import App from './';
```

Comment:
Oof hate this import path, can they import it from its directory instead?

Reply:
since this is the simplest possible example layout I'll just have them import from `./app` and add that to the  little folder layout diagram above